### PR TITLE
suite: debug mgr by default

### PR DESCRIPTION
**Pull request by liewegas**
Like we do the mon and osd.

### CODE REVIEW

File: teuthology/suite/placeholder.py [hunk 1]
```diff
@@ -63,10 +63,13 @@ def _substitute(input_dict, values_dict):
                     'debug mon': 20,
                     'debug ms': 1,
                     'debug paxos': 20},
+                'mon': {
+                    'debug mgr': 20,
+		    'debug ms': 1},
```
Comment (kshtsk) [inline teuthology/suite/placeholder.py]: replace tabs with spaces,

**Comment by kshtsk** [comment]
hm, interesting:
```
teuthology/suite/placeholder.py:62:17: F601 dictionary key 'mon' repeated with different values
teuthology/suite/placeholder.py:66:17: F601 dictionary key 'mon' repeated with different values
```

File: teuthology/suite/placeholder.py [hunk 2]
```diff
@@ -63,10 +63,13 @@ def _substitute(input_dict, values_dict):
                     'debug mon': 20,
                     'debug ms': 1,
                     'debug paxos': 20},
+                'mon': {
```
Comment (kshtsk) [inline teuthology/suite/placeholder.py]: 'mgr'?

Comment (kshtsk) [inline teuthology/suite/placeholder.py]: join with previous record?

Comment (vasukulkarni) [reply teuthology/suite/placeholder.py]: should be mgr, i guess.

**Pull request merged**